### DISCUSSION TITLE
Is it possible to set the crop area size to the loaded image width and height?

### DISCUSSION THREAD

**Discussion by prashanthsonu** [ValentinH/react-easy-crop#509] [Q&A]
Hi, I'm working on cropping the images. The react easy crop is awesome. I'm trying to increase the size of the crop area to the loaded image width and height. Is there any way to do that? 

Also the crop area size should be equivalent to the image height and width if it is rotated 90 degrees every time.

**Comment by ValentinH** [comment]
I'm not sure to understand. Could you please create a sandbox showing the issue?

**Reply by prashanthsonu** [reply]
I mean, I want the full width and height crop.
